doc-content/kogito-docs/src/main/asciidoc/openshift/con_kogito-travel-agency.adoc

```diff
@@ -0,0 +1,40 @@
+[id='con_kogito-travel-agency']
+
+= Travel agency tutorial for {PRODUCT} services on {OPENSHIFT}
+
+The {PRODUCT} https://github.com/kiegroup/kogito-travel-agency-tutorial[travel agency tutorial] in GitHub contains a sequence of example applications with {PRODUCT} services, each of them increasing in complexity. These travel agency applications illustrate many of the advanced configuration options you can use whether you are deploying services locally or on {OPENSHIFT}, such as process persistence with Infinispan, messaging with Kafka, and application data indexing with the Data Index Service.
```

Comment:
Actually, each sequence means a Kogito "version" with capabilities that we had at the time. Of course we increase complexity, but I think we will review this topic to have only one example there. There's a thread on going in the channel. I think we might keep this way, it's just a side note that I thought worth mentioning. cc @evacchi

Reply:
Ah, okay. Good to know. So I'll leave it for now until I hear otherwise. Thanks.